### ISSUE TITLE
orderby.limit changefeed broken in 2.0RC1 Data Explorer

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#3994]
```js
r.tableCreate('t1')
r.table('t1').orderBy({index: "id"}).limit(1).changes()

/ /In a second window:
r.table('t1').insert({id: 0})
```

Expected result: The inserted row should appear on the changefeed. Actually nothing happens.

With a different client the changefeed works fine.

**Comment by deontologician** [comment]
@AtnNn would it be possible for you to take a look at this? I have not familiarized myself with the dataexplorer code in depth.

**Comment by AtnNn** [comment]
I was able to reproduce in other ways as well. It looks like it degrades the cursor into a regular stream and waits for the next 40 results before displaying them as a single batch. I am investigating further.

**Comment by AtnNn** [comment]
This happens because the data explorer needs to be updated to support the changes in #3715

**Comment by AtnNn** [comment]
Fixed in 7be2db9
